src/test/java/space/obminyashka/items_exchange/end2end/UserFlowTest.java

```diff
@@ -162,8 +164,9 @@ void updateUserPassword_shouldReturnException_whenPasswordsSame() throws Excepti
             ignoreCols = {"password", "email", "lastOnlineTime", "updated"})
     void updateUserEmail_shouldGetResponse() throws Exception {
```

Comment:
Don't see any mocking of SendGrid service. Does it mean the real mail would be sent in the scope of the test to a fake user's email?